WDBRemoveThreeAppLimit/helpers.m

```diff
@@ -20,7 +20,10 @@
   char* buf = malloc(PAGE_SIZE*10);
   memset(buf, 'A', PAGE_SIZE*10);
   fwrite(buf, PAGE_SIZE*10, 1, f);
-  //fclose(f);
+  // Clean up resources before returning the file path
```

Comment:
(This isn't actually used - this is taken from Ian Beer's sample code and isn't actually called)